src/constants/abis/keepLPRewards.json

```diff
@@ -0,0 +1,13 @@
+[
```

Comment:
nit: i think you might be able to use `src/constants/abis/erc20.json` instead of adding a new ABI

Reply:
Though the erc20.json would work, for the sake of clarity, we shouldn't use ABIs that don't fully match the contract.